inject/src/main/java/io/micronaut/context/ApplicationContextBuilder.java

```diff
@@ -224,6 +224,16 @@ public interface ApplicationContextBuilder {
      */
     @NonNull ApplicationContextBuilder allowEmptyProviders(boolean shouldAllow);
 
+    /**
+     * Enable strict conversion error checking.
+     * @param strict Whether to enable strict conversion error checking
+     * @return This application
+     * @since 3.9.0
+     */
+    @NonNull default ApplicationContextBuilder strictConversionChecking(boolean strict) {
```

Comment:
why is something related to the request binding including in `ApplicationContextBuilder`? This should be in `HttpServerConfiguraiton` surely?

Reply:
@yawkat can you address this change?